pmm-app/src/pmm-dbaas/components/DBCluster/DBClusterParameters/DBClusterParameters.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC } from 'react';
+import { Spinner, useStyles } from '@grafana/ui';
+import { Messages } from 'pmm-dbaas/DBaaS.messages';
+import { DBClusterParametersProps } from './DBClusterParameters.types';
+import { DBClusterStatus } from '../DBCluster.types';
+import { getStyles } from './DBClusterParameters.styles';
+import { DBClusterConnectionItem } from '../DBClusterConnection/DBClusterConnectionItem/DBClusterConnectionItem';
+import { isClusterChanging } from '../DBCluster.utils';
+
+export const DBClusterParameters: FC<DBClusterParametersProps> = ({ dbCluster }) => {
+  const styles = useStyles(getStyles);
+  const { status } = dbCluster;
+
+  return (
+    <>
+      {!isClusterChanging(dbCluster) ? (
+        <div className={styles.wrapper}>
+          {status && status === DBClusterStatus.ready ? (
+            <>
+              <DBClusterConnectionItem
+                label={Messages.dbcluster.table.parameters.clusterName}
+                value={dbCluster.clusterName}
+                dataQa="cluster-parameters-cluster-name"
+              />
+              <DBClusterConnectionItem
+                label={Messages.dbcluster.table.parameters.cpu}
+                value={dbCluster.cpu}
+                dataQa="cluster-parameters-cpu"
+              />
+              <DBClusterConnectionItem
+                label={Messages.dbcluster.table.parameters.memory}
+                value={`${dbCluster.memory} GB`}
+                dataQa="cluster-parameters-memory"
+              />
+              <DBClusterConnectionItem
+                label={Messages.dbcluster.table.parameters.disk}
+                value={`${dbCluster.disk} GB`}
+                dataQa="cluster-parameters-cpu"
+              />
+            </>
+          ) : (
+            <span className={styles.connectionFailed} data-qa="cluster-connection-failed">
+              -
+            </span>
+          )}
+        </div>
+      ) : (
+        <div className={styles.connectionLoading} data-qa="cluster-connection-loading">
```

Comment:
IMO we should remove the Spinner here. Otherwise we will have the Spinner in two columns and it will be weird. Just keep the "-" when the cluster is changing. Is even simpler.

Reply:
good point, fixed